api/src/Entity/Checklist.php

```diff
@@ -29,28 +29,37 @@
 #[ApiResource(
     operations: [
         new Get(
-            security: 'is_granted("CAMP_COLLABORATOR", object) or is_granted("CAMP_IS_PROTOTYPE", object)'
+            security: 'is_granted("CHECKLIST_IS_PROTOTYPE", object) or 
+                       is_granted("CAMP_IS_PROTOTYPE", object) or 
+                       (null != object.getCamp() and is_granted("CAMP_COLLABORATOR", object))
```

Comment:
`null != object.getCamp()` is now not needed anymore after https://github.com/ecamp/ecamp3/pull/6010 was merged

Same below for Patch and Delete

Reply:
true, thx